### DISCUSSION TITLE
Type Syntax

### DISCUSSION THREAD

**Discussion by fab918** [carbon-language/carbon-lang#2346] [Language design]
Hi, I was looking at the syntax and was very surprised at the use of parenthesis for types, the syntax feel a bit heavy to look at. Kotlin, TS and C++ use <> instead. I have not found any discussion on this subject (I hope I searched well).

It seems strange to break the habits, especially since the function names are capitalised, it gives the impression of a function call.

Before:
```
fn PrintTotalArea(circles: Slice(Circle)) {

var circles: Array(Circle) = ({...});
```

After
```
fn PrintTotalArea(circles: Slice<Circle>) {

var circles: Array<Circle> = ({...});
```

**Comment by geoffromer** [comment]
See [here](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/faq.md#why-arent--and--used-as-delimiters) for an explanation and links to more details.
